### ISSUE TITLE
Panes should be resizable with the mouse

### ISSUE THREAD

**Issue by zadjii-msft** [microsoft/terminal#992]
Probably depends on #<Panes should be resizable with the keyboard> getting done first

Drag a seperator to (recursivly) resize the panes that are seperated by that seperator.

**Comment by carlos-zamora** [comment]
Assigning to myself for now. I imagine it won't be _too_ difficult. If anybody would like to take a crack at it though, feel free to let me know and we could just do a handoff. 😊

**Comment by jonasnordlund** [comment]
One idea I have would be to not even require a user to target a potentially thin separator, but while holding down Alt (to retain the theme with the pane key bindings) and dragging the mouse up and down over one horizontally split pane, resize that one height wise. And the same when moving the mouse left and right over a vertically split pane.

**Comment by moswald** [comment]
It would be nice if, while resizing, a percent or pixel adornment showed up that indicates the new sizes for both panes.

**Comment by wardboumans** [comment]
Any update on this? Its the one thing missing for me.

**Comment by zadjii-msft** [comment]
Nope. We'll make sure to update this thread when there is. In the meantime, might I recommend the Subscribe button?
![image](https://user-images.githubusercontent.com/18356694/91237459-5cbb0c80-e700-11ea-9347-b9b1ec2813b1.png)
That way you'll be notified of any updates to this thread, without needlessly pinging everyone on this thread ☺️

**Comment by mbalc** [comment]
still waiting for this

**Comment by mbalc** [comment]
still waiting for this, almost 4 years passed...

**Comment by zadjii-msft** [comment]
Yep, we haven't had a chance to loop back on this, though we'd love the the contribution if you're interested!

> **Note**
> ## Walkthrough

* Last time I tried this was in https://github.com/microsoft/terminal/compare/main...dev/migrie/f/drag-panes. IIRC, there were some scaling issues with mouse drags in XAML islands back then. I think those are mostly resolved these days. 
* Carlos also tried this again in [`dev/cazamor/drag-panes`](https://github.com/microsoft/terminal/commit/21474792a269bb4a481dd868d93fc7ce281244c2). I don't know how successful that was. 

I don't think it'd be terribly difficult to resurrect the same approach (but base it off `main` since my original branch is nearly 4 years old now.

**Comment by davea38** [comment]
Massively agree with this new feature! Resizing via the keyboard is annoying. Also there are currently too many keyboard shortcuts to remember for an old man such as myself :>

**Comment by bradphelan** [comment]
I think dragging the panes is the wrong UX. It's always a pain the try and get the tiny mouse sitting on the tiny drag bar. The alternative is to make the drag bar thicker and waste terminal space. I propose an alterative. We already have 

<ctrl-mouse-wheel>

to change the font size.  If we add

<ctrl-alt-mouse-wheel> 

and instead of changing font size, it resizes the pane. The location of the mouse itself gives the hint as to where to resize. left, right, up down.

**Comment by anashakt** [comment]
I agree. We need this feature

**Comment by DHowett** [comment]
Punted to 1.23.

**Comment by progressive-galib** [comment]
hi is this working and available in stable version ? how do i drag pane divider in windows terminal ? do i have to press Alt or any other keyboard key while dragging pane divider ?
